app/src/main/java/com.example.weatherapp/data/model/ForecastModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.weatherapp.data.model
+
+data class ForecastModel(
```

Comment:
Let's be careful around the naming of the variables, lets remember we should be using mental mappings, and the models should have as much descriptive naming as possible.

Reply:
You're right i changed it, so now it has a more descriptive name.